README.md

```diff
@@ -1,7 +1,5 @@
 # <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;RAFT: Reusable Accelerated Functions and Tools</div>
 
-[![Build Status](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/raft/job/branches/job/raft-branch-pipeline/badge/icon)](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/raft/job/branches/job/raft-branch-pipeline/)
```

Comment:
Should we consider adding a GHA badge? https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

Reply:
I vote no. Our developers don't spend much time looking at the README, and it looks bad to users when our builds are failing.

Reply:
Agreed, but for a different reason.

The latest build status can already be seen on the repository's home page. And when you switch branches, it's automatically in-sync. So no `update-version.sh` hackery is necessary.

![image](https://user-images.githubusercontent.com/7400326/217407970-aca28410-0005-40b6-8a66-353c0b1bd25b.png)